meta-balena-common/recipes-support/resin-init/resin-init-flasher/resin-init-flasher

```diff
@@ -331,6 +314,7 @@ if [ "$LUKS" = "1" ]; then
         sync "$DM_DEV"
     done
 
+    rm -f "$PASSPHRASE_FILE"
```

Comment:
```suggestion
    rm -f "$PASSPHRASE_FILE" && sync
```

Reply:
Same as above - this is an in-memory temporary file.